product-service-ts/handlers/getProductsById.ts

```diff
@@ -0,0 +1,37 @@
+import { APIGatewayProxyHandler } from 'aws-lambda';
+import products from './products-mock.json';
+
+export const getProductsById: APIGatewayProxyHandler = async (event) => {
+  console.log("getProductsById FN. Event: ", event);
+  console.log("getProductsById FN. Products: ", products);
+
+  try {
+    const { id } = event?.pathParameters || {};
```

Comment:
In case id is not passed you would get `No product with such id ${id}`. But id is required and you should follow fail fast approach and skip search if id is not passed

Reply:
Thanks, done.